python/cuxfilter/themes/light.py

```diff
@@ -1,153 +1,90 @@
-# Layout Head Section style CSS
-# NOTE may move to remote CSS file later
-from bokeh import palettes
-
-
-class Theme:
-    layout_head = """
-    {% extends base %}
-
-    {% block head %}
-    <head>
-        {% block inner_head %}
-        <meta charset="utf-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-        <title>{% block title %}
-        {{ title | e if title else "Panel App" }}{% endblock %}
-        </title>
-        {% block preamble %}{% endblock %}
-        {% block resources %}
-            {% block css_resources %}
-            {{ bokeh_css | indent(8) if bokeh_css }}
-
-            <link rel="stylesheet"
-            href="https://unpkg.com/purecss@1.0.1/build/pure-min.css"
-            integrity="sha384-oAOxQR6DkCoMliIh8yFnu25d7Eq/PHS21PClpwjOTeU2jRSq11vu66rf90/cZr47"
-            crossorigin="anonymous">
-            <link rel="stylesheet"
-            href="https://unpkg.com/purecss@1.0.1/build/grids-responsive-min.css">
-
-            {% endblock %}
-            {% block js_resources %}
-            {{ bokeh_js | indent(8) if bokeh_js }}
-            {% endblock %}
-        {% endblock %}
-        {% block postamble %}
-            <style>
+import pathlib
+import param
 
-                body {
-                    margin:0;
-                    padding:0;
-                }
-
-                nav {
-                    display: inline-block;
-                    vertical-align: top;
-                    width: 18em;
-                    height: calc(100vh - 2em);
-                    padding: 1em;
-                    background-color: #fcfcfc;
-                    color: #a0a0a0;
-                    border-right: 2px solid #fafafa;
-                }
-
-                .nav-title {
-                    font-size: 1.5em;
-                }
+from bokeh.themes import Theme as _BkTheme
+from bokeh import palettes
+from panel.template.theme import DefaultTheme
+from ..layouts.layouts import ReactTemplate
+from ..charts.constants import DATATILE_ACTIVE_COLOR
 
-                .nav-container {
-                    margin-top: 1em;
-                    font-size: 1.04em;
-                }
+ASSETS_PATH = pathlib.Path(__file__).parent / "assets"
```

Comment:
This is defined separately in a few modules now, maybe pull up to one place in `themes/__init__.py` ?

Reply:
Actually, @bryevdv  adding it to the `themes/__init__.py` is throwing a circular import error

Reply:
Added all static paths to `charts/constants.py` where the other constants reside